openedx/stanford/djangoapps/register_cme/forms.py

```diff
@@ -139,22 +146,23 @@ def clean_affiliation(self):
         affiliation = self.cleaned_data['affiliation']
```

Comment:
This method clearly didn't have a unit test. Not necessarily with this fix but can you add one shortly thereafter.